tests/model_options/test_tablespaces.py

```diff
@@ -21,7 +23,9 @@ def sql_for_index(model):
 # We can't test the DEFAULT_TABLESPACE and DEFAULT_INDEX_TABLESPACE settings
 # because they're evaluated when the model class is defined. As a consequence,
 # @override_settings doesn't work, and the tests depend
-class TablespacesTests(TestCase):
+class TablespacesTests(TransactionTestCase):
```

Comment:
What's at stake with the change to `TransactionTestCase` here? Is it just an optimisation? (If so, separate commit?)

Reply:
@carltongibson the `sql_for_table` calls which is defined above have to `SchemaEditor.__enter__` which is now prohibited in a transaction. Using `TransactionTestCase` makes sure we don't wrap tests in a transaction.